app/Singletons/Mods.php

```diff
@@ -135,19 +135,31 @@ public function idsToBitset($ids): int
         );
     }
 
-    public function assertValidExclusivity(int $rulesetId, array $requiredIds, array $allowedIds): bool
+    public function assertValidExclusivity(int $rulesetId, array $modAcronyms): bool
     {
+        while (($modAcronym = array_pop($modAcronyms)) !== null) {
+            $mod = $this->mods[$rulesetId][$modAcronym];
+            $incompatibleIds = $mod['IncompatibleMods'];
```

Comment:
I'm not actually sure why it's done this way originally instead of just

```suggestion
            $incompatibleIds = $this->mods[$rulesetId][$modAcronym]['IncompatibleMods'];
```